pdfarranger/pdfarranger.py

```diff
@@ -951,6 +954,8 @@ def clear_data(self):
         malloc_trim()
 
     def on_quit(self, _action, _param=None, _unknown=None):
+        if self.disable_quit:
+            return True
         if self.export_process and self.export_process.is_alive():
             msg = _('Abort saving and quit?')
```

Comment:
I guess this option could never happen, or am i wrong? Other than this I see no issues.